### ISSUE TITLE
Why the stores I'm adding was deleted?

### ISSUE THREAD

**Issue by bitcoinuser** [lightningnetworkstores/lightningnetworkstores.github.io#112]
Why the stores I'm adding was deleted?

**Comment by pedro-roberto** [comment]
There was a store called DOMORDER that was removed because they stopped accepting lightning. Which store are you talking about?

**Comment by bitcoinuser** [comment]
There are three stores that I added that was removed. It's not this DOMORDER, but I don't remember the names. I will try to find the name of at least one.

**Comment by pedro-roberto** [comment]
Is it CryptAhoy and Bitplaza? I see that someone tried to submit these stores but didn't pay the 1000 satoshis. If it was you, did you forget to put the coupon code?

**Comment by bitcoinuser** [comment]
No, it was not me. The stores I added was others. I can't find the names.

**Comment by pedro-roberto** [comment]
When did you try to add these stores. They were probably never added in the first place as when they are added the server commits to this repository. Anyway, next time I have something to add I will see if there is an issue.

**Comment by bitcoinuser** [comment]
Two stores I added yesterday. The other was some days ago. I saw it listed
and now gone.

**Comment by bitcoinuser** [comment]
Found one of the stores that I already had added: https://farkleshirts.com/

Em qui, 11 de out de 2018 às 14:04, Leonardo <leodocosta@gmail.com>
escreveu:

**Comment by bitcoinuser** [comment]
I'm sure that it was listed on lightningnetworkstores. There are other two
that I don't remember the names that is not showing on the site.

Em qui, 11 de out de 2018 às 14:10, Leonardo <leodocosta@gmail.com>
escreveu:

**Comment by pedro-roberto** [comment]
I see that Farkle shirts was deleted when Joltfun node URI was updated some days ago. When Joltfun did the pullrequest with the node URI change he eliminated Farkle Shirsts and I didn't notice he had done so. It was a human error and not a bug in the code :P Thanks a lot for pointing this out. I will look for similar human errors with the previous description/node URI changes when I have time. 

Thanks again for pointing this out.

**Comment by pedro-roberto** [comment]
Farkle tees was added back, as well as the votes it had before being deleted.
